collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from collections import Counter
+from random import SystemRandom
+import requests
+from bases.FrameworkServices.SimpleService import SimpleService
+
+priority = 25
+update_every = 1
+
+DEFAULT_HOST = '127.0.0.1:19999'
+
+ORDER = [
+    'warning',
+    'critical'
+]
+
+CHARTS = {
+    'warning': {
+        'options': [None, 'Warning', 'warning', 'warning', 'alarms.warning', 'stacked'],
```

Comment:
`warning` units? Probably `boolean` in that case.

Reply:
not sure on this one - i changed it all to be `alarms` units...?